### DISCUSSION TITLE
Sorry, the response matched public code so it was blocked. Please rephrase your prompt.

### DISCUSSION THREAD

**Discussion by VKrishna04** [community/community#76593] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

So i have followed this [https://docs.github.com/en/copilot/configuring-github-copilot/configuring-github-copilot-settings-on-githubcom#enabling-or-disabling-duplication-detection](url) github docs for this bug in VS Code.
![image](https://github.com/community/community/assets/75069043/589f428c-142a-45b1-923e-5caea7794b7d)
like so in [https://github.com/settings/copilot](url) 
![image](https://github.com/community/community/assets/75069043/1690440a-0f9b-4b31-bb63-ca30bb9b69e2)

It says to wait for 30 mins but it is still not working after nearly 1 month.
Thank you.

**Comment by VKrishna04** [comment]
PS I'm not part of any organization nor do I have any organization of my own.

**Comment by Akash1134** [comment, marked as answer]
Hi @VKrishna04 ,

We're sorry about the trouble caused. It seems you might be impacted by a bad deploy that happened [yesterday](https://github.com/microsoft/vscode-copilot-release/issues/583).  

Please restart VSCode and logout/login to the Copilot extension. This issue has already been resolved. 

Thanks!

**Reply by VKrishna04** [reply]
Thanks a lot now its working in 0.10.2 in VS Code after `reloading with clearing cache` and then its answering normally again.
